src/components/SearchResults.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from "react";
+import "./SearchResults.css";
+import moment from "moment";
+const SearchResults = props => {
+  const [isSelected, setIsSelected] = useState([]);
+  const setClassName = id => {
```

Comment:
Well done on getting this function to work, the logic is good! However, you could make it a bit simpler by using a boolean check for `isSelected` rather than a String